### ISSUE TITLE
Box86 on SWITCH L4T

### ISSUE THREAD

**Issue by hwanghyunseo** [ptitSeb/box86#503]
i have switchroot ubuntu. its l4t ubuntu actions file is how to use? for arm64 device?

**Comment by hwanghyunseo** [comment]
actions file:https://github.com/ptitSeb/box86/actions

**Comment by Seas0** [comment]
**In short**, box86 on SWITCH L4T is **NOT** a favorable consideration for now.
First, notice that the box86 project current does **NOT** include 32-bits to 64-bits translation, and is useless on a pure 64-bits OS. 
Second, As for SWITCH L4T, my assumption is that just like the case of Nvidia Jetson Nano, as the Ubuntu L4T on SWITCH is actually transplanted from Nano or TX 1. In the [document](https://github.com/ptitSeb/box86/blob/master/docs/COMPILE.md#testing)  it is said that 
_Note about devices with Tegra X1 and newer.

Nvidia don't provide armhf libraries for their GPU drivers at this moment so there is no special variable to compile it for them as it would be missleading for many people. If you still want to use it wihout GPU acceleration, building it with RPI4 configuration should work just fine. As instalation of Mesa can break Nvidia driver, safest option is usage of chroot._

So in conclusion, box86 on SWITCH L4T is not a good idea until Nvidia release the armhf version of their proprietary graphical drivers, which I don't actually think is possible any time soon based on a post on their official community. [Original post](https://forums.developer.nvidia.com/t/add-gpu-libs-32bits-support-in-the-future/123809)

**Comment by Seas0** [comment]
Also, you may change the issue title to _Box86 on SWITCH L4T_ for a better description of your problem
